main/php_globals.h

```diff
@@ -154,7 +154,7 @@ struct _php_core_globals {
 	char *mail_log;
 	bool mail_x_header;
 	bool mail_mixed_lf_and_crlf;
-	char *mail_cr_lf_mode;
+	zend_string *mail_cr_lf_mode;
```

Comment:
You also made a struct hole by interrupting the booleans, if you move it to right below mail_log, you get rid of the hole.

Reply:
Ah right, PR updated, thanks!